blockchain/network.py

```diff
@@ -700,22 +700,22 @@ def get_unsent_transfer_ids(self, transfer_to=None, verification_id=None):
 
         return unsent_transfer_ids
 
-    def transfer_data(self, received, unreceived):
+    def transfer_data(self, transfer_to, received, unreceived):
         """ mark verifications as sent and return unsent verifications """
         verifications = []
         node_unreceived = []
         all_guids = received + unreceived
 
         # retrieve ids that calling node is claiming as unreceived
         for guid in unreceived:
-            node_unreceived += self.get_unsent_transfer_ids(transfer_to=None, verification_id=guid)
+            node_unreceived += self.get_unsent_transfer_ids(transfer_to=transfer_to, verification_id=guid)
```

Comment:
can we change the `get_unsent_transfer_ids` api to only use the `transfer_to` param? I don't see why we need the `verification_id` param.